src/deprecations.jl

```diff
@@ -372,3 +372,8 @@ end
 
 getdim(args...) = error("`Ferrite.getdim` is deprecated, use `getrefdim` or `getspatialdim` instead")
 getfielddim(args...) = error("`Ferrite.getfielddim(::AbstractDofHandler, args...) is deprecated, use `n_components` instead")
+
+function default_interpolation(cell::AbstractCell)
```

Comment:
The deprecation is on the wrong dispatch. This should be `default_interpolation(cell_type::Type{<:AbstractCell)`or something. See the remaining code in the PR.

Reply:
Thanks, the docstring is also wrong!
```julia
help?> geometric_interpolation
search: geometric_interpolation

  geometric_interpolation(::AbstractCell)::Interpolation

  Each AbstractCell type has a unique geometric interpolation describing its geometry. This function returns that
  interpolation.
```